### PR TITLE
[IMP] theme_kea: revamp theme

### PR DESCRIPTION
**Pull request by mano-odoo**
small design fixes.

task-4178076
Part of task-4177975

### CODE REVIEW

File: theme_kea/views/snippets/s_image_punchy.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_picture" inherit_id="website.s_picture">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc1" remove="o_cc2" separator=" "/>
+    </xpath>
+</template>
```
Comment (bso-odoo) [inline theme_kea/views/snippets/s_image_punchy.xml:9]: This looks fishy: the `s_picture` template is already patched in `s_picture.xml`.

File: theme_kea/views/snippets/s_color_blocks_2.xml [hunk 2]
```diff
@@ -10,27 +10,43 @@
     <xpath expr="//h2" position="before">
         <div class="o_we_bg_filter bg-black-50"></div>
     </xpath>
+    <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" " />
+        <attribute name="style" add="text-align: center; !important" separator=" " />
```
Comment (bso-odoo) [inline theme_kea/views/snippets/s_color_blocks_2.xml:15]: Across the whole commit: isn't `!important` supposed to be before the semi-colon ???

**Review by bso-odoo — COMMENTED** [review]
Are we sure about this shape connection in the generated home page ?
image
It works great in the generated "About Us" tough - maybe the shape should be on the "about us"-specific template ?

**Comment by mano-odoo** [comment]
> maybe the block should have the color of the previous shape as background.

That was the goal but my distraction issue in `s_image_punchy` was probably breaking the o_cc3 applied on `s_picture`, sorry about this 🙇 

I applied the changes req. above + replace `o_cc1` for `o_cc5` on `s_image_punchy` it felt more inline with the theme 😄

**Comment by bso-odoo** [comment]
FYI from runbot: `o_cc5` seems present twice in some `s_cover` variants.
```
AssertionError: ["Using 'Kea Theme', view 'website.new_page_template_sections_landing_0' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'oe_img_bg', 'o_full_screen_height', 'o_cc5', 'pt256', 'pb256']", "Using 'Kea Theme', view 'website.new_page_template_sections_landing_2' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_cc5', 'o_half_screen_height']", "Using 'Kea Theme', view 'website.new_page_template_sections_pricing_2' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_cc5', 'o_half_screen_height']", "Using 'Kea Theme', view 'website.new_page_template_sections_gallery_4' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_cc5', 'o_half_screen_height']", "Using 'Kea Theme', view 'website.new_page_template_sections_landing_4' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_cc5', 'o_half_screen_height']", "Using 'Kea Theme', view 'website.new_page_template_sections_about_mini' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'oe_img_bg', 'o_cc5', 'pt40', 'pb40', 'o_half_screen_height']", "Using 'Kea Theme', view 'website.configurator_homepage_s_cover' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_full_screen_height', 'o_cc5']", "Using 'Kea Theme', view 'website.configurator_s_cover' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_full_screen_height', 'o_cc5']", "Using 'Kea Theme', view 'website.snippets' contains duplicate classes: ['s_cover', 'o_cc', 'o_cc5', 'pt232', 'pb232', 'oe_img_bg', 'o_full_screen_height', 'o_cc5']"] is not false : No error should have been collected
```

**Comment by mano-odoo** [comment]
Oopsie, it's due to my rebase, the `o_cc5` is now on the default `s_cover` template 🤦 Sorry for the trouble I'll re-ask for review once it's fully ready

**Comment by bso-odoo** [comment]
@robodoo r+

**Pull request closed without merging**